test/esm/esm.mjs

```diff
@@ -1,13 +1,37 @@
 import t from 'tap'
 import Fastify from '../../fastify.js'
+import DefaultFastify, { fastify } from '../../fastify.mjs'
```

Comment:
`import Fastify from '../../fastify.js'` is not needed

Reply:
Actually it checks that CJS context is still usable and it is the same as the native ESM context. I know, it is a mess XD